### DISCUSSION TITLE
v95.0.4638.79 release is missing ARM apks

### DISCUSSION THREAD

**Discussion by elcste** [bromite/bromite#1595] [General]
I'm not seeing ARM apk files for the recent [v95.0.4638.79 release](https://github.com/bromite/bromite/releases/tag/95.0.4638.79). I do see ARM64 and x86.

Thanks for all your work on this project!

**Comment by allenluce** [comment]
I'm also not seeing a Chromium release for this tag, https://chromium.googlesource.com/chromium/src/+/refs/tags/95.0.4638.79 gives me a 404.

**Reply by csagan5** [reply]
Does not correspond to a Chromium release, `95.0.4638.78` was used.

**Comment by csagan5** [comment]
The missing ARM files are now released.
